tools/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/ApplicationInfo.scala

```diff
@@ -513,6 +520,15 @@ class ApplicationInfo(
       } else {
         logInfo("No Plan node accums Found. Create an empty Plan node accums DataFrame.")
       }
+
+      // For unsupportedSQLPlanDF
+      allDataFrames += (s"unsupportedSQLplan_$index" -> unsupportedSQLplan.toDF)
+      if (unsupportedSQLplan.nonEmpty) {
+        logInfo(s"Total ${unsupportedSQLplan.size} Plan node accums for appID=$appId")
```

Comment:
"Plan node accums" is copy and paste from above, change it to something like unsupported ops